tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,8 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert len(vendor.inventory) == 3
```

Comment:
This test is checking if get_by_category returns no items in the case where there are no matching items in the inventory.  The assert needs to check the state of items in order to correctly verify that the function did what was expected:
```suggestion
    assert len(vendor.inventory) == 3
    assert len(items) == 0
```